themes-default/slim/views/viewlogs.mako

```diff
@@ -77,10 +77,14 @@ const startVue = () => {
                     min_level: minLevel,
                     log_filter: filter,
                     log_period: period,
-                    log_search: search
+                    log_search: search,
+                    limit: 100
```

Comment:
@sharkykh we need a way to pass a time as a limit instead of just a count of items.